app/jinja_filters.py

```diff
@@ -201,6 +203,26 @@ def format_number_to_alphabetic_letter(number):
     return ''
 
 
+@blueprint.app_template_filter()
+def min_value(a, b):
+    try:
+        return min(i for i in [a, b] if i is not None)
+    except (TypeError, UndefinedError):
+        msg = ('Cannot determine minimum of incompatible types '
+               'min({}, {})'.format(a.__class__, b.__class__))
+        raise Exception(msg)
+
+
+@blueprint.app_template_filter()
+def max_value(a, b):
+    try:
+        return max(i for i in [a, b] if i is not None)
+    except (TypeError, UndefinedError):
+        msg = ('Cannot determine maximum of incompatible types '
+               'max({}, {})'.format(a.__class__, b.__class__))
+        raise Exception(msg)
```

Comment:
@davecharles Can we pass in the min or max here then have this all in same method, as the code is very similar